src/routes/Profile.jsx

```diff
@@ -0,0 +1,78 @@
+// React Imports
+import React, { useEffect, useState } from 'react';
+import { useLocation } from 'react-router-dom';
+// Inrupt Imports
+import { useSession } from '@inrupt/solid-ui-react';
+// Material UI Imports
+import Box from '@mui/material/Box';
+import Link from '@mui/material/Link';
+import Typography from '@mui/material/Typography';
+// Utility Imports
+import { fetchProfileInfo } from '../model-helpers';
+// Component Inputs
+import ProfileInputField from '../components/Profile/ProfileInputField';
+
+/**
+ * Profile Page - Page that displays the user's profile card information and
+ * allow users to edit/update them on PASS
+ *
+ * @memberof Pages
+ * @name Profile
+ * @returns {React.JSX.Element} The Profile Page
+ */
+const Profile = () => {
+  const location = useLocation();
+  const { session } = useSession();
+
+  localStorage.setItem('restorePath', location.pathname);
+
+  const [profileName, setProfileName] = useState(null);
+  const [organization, setOrganization] = useState(null);
+
+  const fetchProfileData = async () => {
+    const profileObject = await fetchProfileInfo(session);
+
+    setProfileName(profileObject.profileInfo.profileName);
+    setOrganization(profileObject.profileInfo.organization);
+  };
+
+  useEffect(() => {
+    if (profileName === null) {
+      fetchProfileData();
+    }
+  }, [profileName]);
+
+  return (
+    <Box sx={{ display: 'flex', flexDirection: 'column', gap: '20px', padding: '30px' }}>
+      <Box sx={{ display: 'flex', flexDirection: 'column', gap: '20px' }}>
+        <Typography sx={{ fontWeight: 'bold', fontSize: '18px' }}>Profile Information</Typography>
+
+        <Typography>
+          User WebId:{' '}
+          <Link href={session.info.webId} target="_blank" rel="noreferrer">
+            {session.info.webId}
+          </Link>
+        </Typography>
+
+        {/* TODO: Refactor/optimize the form below once we have more input */}
+        {/* fields to update profile for */}
+        <Box style={{ display: 'flex', flexDirection: 'column', gap: '15px' }}>
+          <ProfileInputField
```

Comment:
Ultimately, I think from a UX perspective I'd prefer the whole thing be a single form instead of a form for each input and an edit button for each input.   I think 1 edit button that opens all inputs to make changes and a single "SAVE CHANGES" button to submit the form and run the functions.    If this is agreed upon direction, could be a future PR to address/optimize.

Reply:
Think we could have this opened as a discussion on an issue.

Reply:
I think Scott's right, that would be a much simpler and more intuitive form. I'd rather not introduce code into the system that immediately creates issues. If there are follow-up tasks needed, then I think it's a good idea to state explicitly in advance what those follow-up tasks are and why they aren't being done here.

It's ok if this PR doesn't finish the feature. I think it's perfectly fine to say something like "this is the first of 3 planned PRs. The first PR does these things, the second PR will build off that with these other things, and the third PR will package everything up in a nice tidy bow" I just want to know what those PRs are, and what the ultimate goal of this system is. What do you expect this system to look like when everything is done?

Reply:
Basically, what I'm expecting when this page is done, a user can view and edit their profile card on PASS. How it’ll be implemented I felt a singular input approach seemed appropriate, which was why the PR was structured like this.

But I could switch this up to have it be just one big form.

Reply:
If I'm having it switched up to one form, I think we could keep thinking in one PR. Any potential extentions to this will just be adding input fields to the form.